<launch>
	<!-- Startup the camera as a robot listening to commands and delivering images -->
	<node name="cam_actuator_node" pkg="logitech_cam" type="cam_actuator_node" />
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" />
	<node name="image_view" pkg="image_view" type="image_view">
		<remap from="image" to="/usb_cam/image_raw" />
	</node> 
</launch>